tests/phpunit/api/v3/ContributionPageTest.php

```diff
@@ -1416,4 +1416,66 @@ public function addPriceFields(&$params) {
     );
   }
 
+  /**
+   * Test Tax Amount is calculated properly when using PriceSet with Field Type = Text/Numeric Quantity
+   */
+  public function testSubmitContributionPageWithPriceSetQuantity() {
```

Comment:
it looks good - I'm glad you found a way to actually test the page rather than just a true unit test as I think it's more reassuring